app/views/play/modal/PlayItemsModal.coffee

```diff
@@ -212,6 +214,45 @@ module.exports = class PlayItemsModal extends ModalView
       @$el.one 'click', (e) ->
         button.removeClass('confirm').text($.i18n.t('play.unlock')) if e.target isnt button[0]
 
+  onSellButtonClicked: (e) ->
+    e.stopPropagation()
+    button = $(e.target).closest('button')
+    item = @idToItem[button.data('item-id')]
+    
+    # Prevent further action if the item cannot be sold
+    if !item.sellable()
+      return
+    
+    if button.hasClass('confirm')
+      # Sell logic here
+      sale = Sale.makeFor(item)
+      sale.save()
+    
+      #- set local changes to mimic what should happen on the server...
+      sales = me.get('sales') ? {}
+      sales.items ?= []
+      sales.items.push(item.get('original'))
+    
+      purchased = me.get('purchased') ? {}
+      purchased.items ?= []
+      _.pull(purchased.items, item.get('original'))
+      me.set('purchased', purchased)
+    
```

Comment:
In the future POST and DELETE /db/user/:handle/sold endpoints, this logic should go there (since it should use a mongodb update rather than save). Then that endpoint can return the updated user object and the client can just re-render with me already up to date.